jwt-server/spring/jwt-spring-core/src/main/java/org/entur/jwt/spring/JwtAutoConfiguration.java

```diff
@@ -1,39 +1,11 @@
 package org.entur.jwt.spring;
 
-import static org.springframework.security.config.http.SessionCreationPolicy.STATELESS;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import java.util.Set;
-
 import org.entur.jwt.spring.actuate.JwksHealthIndicator;
-import org.entur.jwt.spring.filter.DefaultJwtDetailsMapper;
-import org.entur.jwt.spring.filter.DefaultJwtPrincipalMapper;
-import org.entur.jwt.spring.filter.JwtAuthenticationExceptionAdvice;
-import org.entur.jwt.spring.filter.JwtAuthenticationFilter;
-import org.entur.jwt.spring.filter.JwtAuthorityMapper;
-import org.entur.jwt.spring.filter.JwtDetailsMapper;
-import org.entur.jwt.spring.filter.JwtPrincipalMapper;
+import org.entur.jwt.spring.filter.*;
```

Comment:
Kanskje smak og behag, men jeg tenker at enkeltvis import er like bra eller bedre.

Reply:
Enig, men men autoformatereren nekter. Jeg tror alle klassene er med i de to pakkene uansett, så teknisk sett ikke et problem.